karma.conf.js

```diff
@@ -0,0 +1,65 @@
+let webpackConfig = require('./webpack.config.js');
+//Need to remove externals otherwise they won't be included in test
+delete webpackConfig.externals;
+// Need to define inline source maps when using karma
+webpackConfig.devtool = 'inline-source-map';
+
+const isWindows = /^win/.test(process.platform);
+const isMacOS = /^darwin/.test(process.platform);
+// Create custom launcher in case running with Travis
+const customLaunchers = {
+  Chrome_travis_ci: {
+    base: 'Chrome',
+    flags: ['--no-sandbox', '--autoplay-policy=no-user-gesture-required']
+  }
+};
+
+const launchers = {
+  Chrome_browser: {
+    base: 'Chrome',
+    flags: ['--no-sandbox', '--autoplay-policy=no-user-gesture-required']
+  }
+};
+
+module.exports = function (config) {
+  let karmaConf = {
+    logLevel: config.LOG_INFO,
+    browserDisconnectTimeout: 30000,
+    browserNoActivityTimeout: 60000,
+    customLaunchers: launchers,
+    browsers: ['Chrome_browser', 'Firefox'],
```

Comment:
I think you can set Safari as a browser and add the relevant tests [karma safari luncher](https://www.npmjs.com/package/karma-safari-launcher)